readme.md

```diff
@@ -323,13 +337,13 @@ If we run these commands, we're dropping our entire database and creating a bran
 * Why is this potentially super dangerous?
 
 Another (wrong) way we can do this is by using `rails db:rollback`.
-* To undo a single migration, run `rails db:rollback` in the Terminal.
+* To undo a single migration run `rails db:rollback` in the Terminal
 * Be careful though -- this might destroy data! Whatever columns or tables that were created by that migration will now be gone.  
 
```

Comment:
Yeah I see what you mean. Will stick with "might" only because it doesn't happen 100% of the time.